Sources/W3CTraceContext/TraceContext.swift

```diff
@@ -30,6 +30,23 @@ public struct TraceContext: Equatable {
         self.state = state
     }
 
+    /// When `true`, the least significant bit (right-most), denotes that the caller may have recorded trace data.
```

Comment:
No need to talk about the bit in the first sentence? We can keep the info though below if you want?

This is a copy from the spec right?
Maybe we can have a short info for us and below keep the copy from the spec?